src/Compilers/Core/Desktop/AnalyzerFileReference.cs

```diff
@@ -208,13 +213,12 @@ internal void AddAnalyzers(ImmutableArray<DiagnosticAnalyzer>.Builder builder, s
                 // If there are no analyzers, don't load the assembly at all.
                 if (!analyzerTypeNameMap.ContainsKey(language))
                 {
-                    this.AnalyzerLoadFailed?.Invoke(this, new AnalyzerLoadFailureEventArgs(AnalyzerLoadFailureEventArgs.FailureErrorCode.NoAnalyzers, null, null));
```

Comment:
I actually think it at least deserves to be a Warning. Skipping running analyzers when provided with a _true_ analyzer assembly with analyzers, is a bad false positive about the project being clean against analyzers. Keeping it as info is likely going to cause it to never show up on the console output for command line builds, and not show in the output window by default.